src/main/lrsql/system/webserver.clj

```diff
@@ -109,6 +110,18 @@
                 :sec-head-download     sec-head-download
                 :sec-head-cross-domain sec-head-cross-domain
                 :sec-head-content      sec-head-content}}))
+        routes (add-openapi-route
+                routes
+                {:lrs lrs
+                 :head-opts {:sec-head-hsts         sec-head-hsts
+                             :sec-head-frame        sec-head-frame
+                             :sec-head-content-type sec-head-content-type
+                             :sec-head-xss          sec-head-xss
+                             :sec-head-download     sec-head-download
+                             :sec-head-cross-domain sec-head-cross-domain
+                             :sec-head-content      sec-head-content}
```

Comment:
maybe let-bind the head-opts map above so you can just refer to it here and at the other site above

Reply:
done